doc/whats_new.rst

```diff
@@ -22,6 +22,8 @@ BUG
 
     - Fix computation of AR coefficients across channels in :func:`mne.time_frequency.fit_iir_model_raw` by `Eric Larson`_
 
+    - Fixed maxfilter channel names extra space bug :func:``maxwell.py`` and func:``test_maxwell.py`` are updated by `Sheraz Khan`_
```

Comment:
single-backticks and complete function name (at least to the submodule level) for linking as:
```
:func:`mne.preprocessing.maxwell_filter`
```
No need to mention changes to `test_maxwell.py`